modelql-core/src/commonMain/kotlin/org/modelix/modelql/core/ContextValue.kt

```diff
@@ -13,6 +13,7 @@
  */
 package org.modelix.modelql.core
 
+@Deprecated("use org.modelix.kotlin.utils.ContextValue from org.modelix:kotlin-utils")
```

Comment:
All new deprecations should probably use the [ReplaceWith](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-deprecated/-init-.html) argument.

Reply:
The API is different and an automated replacement not possible.